app/controllers/events_controller.rb

```diff
@@ -5,6 +5,8 @@ class EventsController < ApplicationController
   before_action :authorize_event_author, only: [:update, :destroy]
 
   def index
+    @events = @project.events
+    @project_id = @project.id
```

Comment:
yup, and I had exactly this code (with @project) in the previous commit but you said not to use @project because it's mixing concerns so I switched to @project_id instead